hazelcast/proxy/vector_collection.py

```diff
@@ -90,6 +197,22 @@ def search_near_vector(
         include_vectors: bool = False,
         limit: int = -1
     ) -> Future[List[SearchResult]]:
+        """Returns the Documents closest to the given vector.
+
+        The search is performed using distance metric set when
+        creating the vector index.
+
+        Args:
+            vector: The vector to be used as the reference.
+                It must have the same dimension as specified when creating the vector index.
+            include_value: Return value attached to the Document.
+            include_vectors: Return vectors attached to the Document.
+            limit: Limit the maximum number of Documents returned.
+                If not set, the server default is used.
```

Comment:
there is no server default, I removed `-1` default from Python client in https://github.com/hazelcast/hazelcast-python-client/pull/679

Reply:
I think we should have a server default.

Reply:
I've set it to 10 by default for now.